sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/converter/KotlinTree.kt

```diff
@@ -36,7 +36,8 @@ data class KotlinSyntaxStructure(val ktFile: KtFile, val document: Document, val
     companion object {
         @JvmStatic
         fun of(content: String, environment: Environment, inputFile: InputFile): KotlinSyntaxStructure {
-            val psiFile: KtFile = environment.ktPsiFactory.createFile(normalizeEol(content))
+
+            val psiFile: KtFile = environment.ktPsiFactory.createFile(inputFile.filename(), normalizeEol(content))
```

Comment:
Looks ok, I am just not sure if we want to have just a filename or a path to the file?

Reply:
Yea of course, if in doubt that can be a lot more helpful